modules/examples/todolist/README.md

```diff
@@ -0,0 +1,24 @@
+# TodoList RPC example
+
+## Execution
+
+Running the server:
+
+```bash
+sbt example-todolist-server/run
+```
+
+Running the client:
+
+```bash
+sbt example-todolist-client/run
+```
+
+[comment]: # (Start Copyright)
+# Copyright
+
+Freestyle is designed and developed by 47 Degrees
+
+Copyright (C) 2017 47 Degrees. <http://47deg.com>
```

Comment:
2018?

Reply:
Copy/paste :smile:

Reply:
Actually, the license should be generated by sbt-org-policies, and it should be 2017-2018 though. But nevermind, it can be updated later on.

Reply:
I updated it in the next commit: a1eb6bc877fbaf95d2d107fa26f356ae2a78db0d